doc/sphinx/source/analysis.txt

```diff
@@ -0,0 +1,34 @@
+=================
+ Analysis modules
+=================
+
+.. versionadded:: 0.8.0
+The Analysis modules help in the implementation analyses of
+MDPOW simulations. To simplify the process of analyzing
+a collection of systems generated by a free energy simulation
+the objects in mdpow.ensemble allow for a molecule directory's
+systems to be loaded into `MDAnalysis <https://mdanalysis.org>`_
+Universes and be analyzed as a group.
+
+Ensemble Analysis Framework
+===========================
+
+EnsembleAnalysis
+----------------
+
+A framework for analyzing MDPOW simulations loaded into ensemble objects.
+
+.. autoclass:: mdpow.analysis.ensemble.EnsembleAnalysis
+    :members:
+
+    .. automethod:: _prepare_ensemble
+
+    .. automethod:: _prepare_universe
+
+    .. automethod:: _single_universe
+
+    .. automethod:: _single_frame
+
+    .. automethod:: _conclude_ensemble
+
+    .. automethod:: _conclude_universe
```

Comment:
all of this should be in `analysis/ensemble.txt`

Reply:
done